langchain/callbacks/manager.py

```diff
@@ -398,6 +402,55 @@ async def on_tool_error(
         )
 
 
+def _configure(
+    callback_manager_cls: Type[CallbackManager] | Type[AsyncCallbackManager],
+    inheritable_handlers: Optional[
+        Union[BaseCallbackManager, List[BaseCallbackHandler]]
+    ] = None,
+    local_handlers: Optional[
+        Union[BaseCallbackManager, List[BaseCallbackHandler]]
+    ] = None,
+    verbose: bool = False,
+) -> Optional[BaseCallbackManager]:
+    callback_manager: Optional[BaseCallbackManager] = None
+    if inheritable_handlers or local_handlers:
+        if isinstance(inheritable_handlers, list) or not inheritable_handlers:
+            callback_manager = callback_manager_cls(
+                handlers=inheritable_handlers, inheritable_handlers=inheritable_handlers
+            )
+        else:
+            callback_manager = inheritable_handlers
+        callback_manager = copy.deepcopy(callback_manager)
+        local_handlers_ = (
+            local_handlers
+            if isinstance(local_handlers, list)
+            else (local_handlers.handlers if local_handlers else [])
+        )
+        [callback_manager.add_handler(handler, False) for handler in local_handlers_]
+
+    tracing_enabled = os.environ.get("LANGCHAIN_TRACING") is not None
+    if verbose or tracing_enabled:
+        if not callback_manager:
+            callback_manager = callback_manager_cls([])
+        std_out_handler = StdOutCallbackHandler()
```

Comment:
minor nit: may as well define this in the block below since it only seems used there